client-grpc/src/lib.rs

```diff
@@ -1,4 +1,20 @@
-//! Exposes svc-telemetry Client Functions
+//! <center>
+//! <img src="https://github.com/Arrow-air/tf-github/raw/main/src/templates/doc-banner-services.png" style="height:250px" />
+//! </center>
+//! <div align="center">
+//!     <a href="https://github.com/Arrow-air/lib-ccsds/releases">
+//!         <img src="https://img.shields.io/github/v/release/Arrow-air/lib-ccsds?include_prereleases" alt="GitHub release (latest by date including pre-releases)">
+//!     </a>
+//!     <a href="https://github.com/Arrow-air/lib-ccsds/tree/main">
+//!         <img src="https://github.com/arrow-air/lib-ccsds/actions/workflows/rust_ci.yml/badge.svg?branch=main" alt="Rust Checks">
+//!     </a>
+//!     <a href="https://discord.com/invite/arrow">
+//!         <img src="https://img.shields.io/discord/853833144037277726?style=plastic" alt="Arrow DAO Discord">
+//!     </a>
```

Comment:
some links seem to be pointing to lib-ccsds

Reply:
nice catch!

https://github.com/Arrow-air/svc-telemetry/pull/5/commits/26dbe0229d09340013fc1986e94dade4d3a69c15